Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1043,6 +1046,24 @@ def get_full_report_command(
         ]
     return command_results, is_command_finished
 
+def validate_sandbox_report(report_resources: list[dict[str, Any]]) -> str:
+    """This function checks for any error messages in the sandbox report.
+
+    Args:
+        report_resources (list[dict[str, Any]]): Report resources of the report. They hold the data about
+        any error messages returned from the report.
+
+    Returns:
+        str: The error message found in the sandbox, or an empty string if no error is found.
+    """
+    for resource in report_resources:
+        resource_id = resource.get('id')
```

Comment:
```suggestion
        resource_id = resource['id']
```
Id should always be present.